### PR TITLE
BSol Price EA

### PR DESCRIPTION
**Pull request by cds95**
This PR implements the BSol price EA.

### CODE REVIEW

**Review by austinborn — COMMENTED** [review]
I know this is still a draft but I figured I can note these down ahead of time.

File: packages/composites/bsol-price/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# @chainlink/example-composite-adapter
+
+## 1.0.26
+
```
[the remark this answers is not in the document]
Comment (cds95) [reply packages/composites/bsol-price/CHANGELOG.md:4]: I think so.  I think that this was generated for me when I ran the command to create a new EA.  We should probably remove it from there as well in the future.

File: packages/composites/bsol-price/package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@chainlink/bsol-price-adapter",
+  "version": "1.0.26",
```
[the remark this answers is not in the document]
Comment (cds95) [reply packages/composites/bsol-price/package.json:3]: Yeah we should.  Again this is something that we should fix in the script to generate new EAs.

[the remark this answers is not in the document]
Comment (cds95) [reply packages/composites/bsol-price/package.json:3]: Good catch!

File: packages/sources/solana-view-function/schemas/env.json [hunk 3]
```diff
@@ -1,15 +1,27 @@
 {
   "$id": "https://external-adapters.chainlinklabs.com/schemas/solana-view-function-adapter.json",
-  "title": "@chainlink/solana-view-function-adapter env var schema",
+  "title": "SOLIDO_CONTRACT_VERSION",
```
Comment (austinborn) [inline packages/sources/solana-view-function/schemas/env.json:3]:
```suggestion
  "title": "Chainlink External Adapter for Solana-view-function",
  "description": "The Solana view function adapter can be used to query account information from the Solana blockchain",
```

**Review by austinborn — APPROVED** [review]
If you can fix that one schema title, the rest looks great!

**Review by austinborn — DISMISSED** [review]

**Review by austinborn — APPROVED** [review]

**Pull request merged**